src/api/getTrendingArticles.js

```diff
@@ -12,7 +12,7 @@ export const getTrendingArticles = (lang, country) => {
   const url = buildMwOrgApiUrl(params)
   return cachedFetch(url, data => {
     const page = data.query.pages[0]
-    if (page.missing) {
+    if (page && page.missing) {
```

Comment:
I don't see a good reason to have the extra guard here, what if `data.query` is undefined? 

a better solution would be having try catch form and failureCallback in the promise, but we don't apply this to all the api layer.

if the api changes the response object, we will need to update it too.

Reply:
so I changed L14 to
```const page = data.query.pages[-1]```
and I get this error
![image](https://user-images.githubusercontent.com/5482105/116541677-ad17e900-a8e3-11eb-9f45-0161d5a8205a.png)

I feel like this is a rabbit hole though, since this won't probably happen in prod?
Shall we just ignore this?

Reply:
Fine by me